docs/content/howto/open-any-file.md

```diff
@@ -14,9 +14,13 @@ All these file loading methods support loading a single file, many files at once
 
 ⚠ Drag-and-drop of folders does [not yet work](https://github.com/rerun-io/rerun/issues/4528) on the web version of the Rerun Viewer ⚠
 
-The following file types have built-in support in the Rerun Viewer:
+As for the SDK, check out the `log_file_from_path` and `log_file_from_contents` methods ([C++](https://ref.rerun.io/docs/cpp/stable/classrerun_1_1RecordingStream.html#SOME_HASH_TBD?speculative-link), [Python](https://ref.rerun.io/docs/python/stable/common/other_classes_and_functions/#rerun.log_file_from_path?speculative-link), [Rust](https://docs.rs/rerun/latest/rerun/struct.RecordingStream.html#method.log_file_from_path)) and the associated examples ([C++](https://github.com/rerun-io/rerun/blob/main/examples/cpp/log_file/main.cpp), [Python](https://github.com/rerun-io/rerun/blob/main/examples/python/log_file/main.py), [Rust](https://github.com/rerun-io/rerun/blob/main/examples/rust/log_file/src/main.rs)).
```

Comment:
I know it's more work but a code switcheroo box for a minimal example would be nicer. I'm not sure if it's otherwise entirely clear for a newcomer what this is even about

Reply:
Also, I'd go about it more along the lines of:

```suggestion
To log the contents of a file from the SDK you can use the `log_file_from_path` and `log_file_from_contents` methods 
```